tools/mason/MasonUpdate.chpl

```diff
@@ -45,13 +46,33 @@ The current resolution strategy for Mason 0.1.0 is the IVRS as described below:
 */
 
 private var failedChapelVersion: list(string);
-
+private var update = false;
 //
 // Temporary passthrough transforming array to list to appease the compiler.
 //
-proc UpdateLock(args: [?d] string, tf="Mason.toml", lf="Mason.lock") {
```

Comment:
We don't use these args anywhere else anymore?

Reply:
I believe only `mason update` from `mason.chpl` called this function. Other than that, I didn't get any error else where

Reply:
OK thanks!